src/ucx_van.h

```diff
@@ -384,11 +386,11 @@ class UCXEndpointsPool {
     UCXEndpointsPool *p = reinterpret_cast<UCXEndpointsPool*>(arg);
     uint64_t id         = *(reinterpret_cast<uint64_t*>(data));
 
-    UCX_LOG_BASE(3, p->my_node_, "ep create, got AM REPLY node id " << id
+    UCX_LOG_BASE(1, p->my_node_, "ep create, got AM REPLY ep id " << id
                  << " ep " << reply_ep << " eps size " << p->client_eps_.size());
     for(auto it = p->client_eps_.begin(); it != p->client_eps_.end(); ++it)
     {
-        UCX_LOG_BASE(3, p->my_node_, "Key " << it->first );
+        UCX_LOG_BASE(1, p->my_node_, "Key " << it->first );
```

Comment:
i'd remove this one from master at all, because it may flood the logs when there are many workers and servers used.
add just for the debugging of "ep not connected" failure

Reply:
ok